tests/beacon/types/test_candidate_pow_receipt_root_record.py

```diff
@@ -0,0 +1,13 @@
+from eth.beacon.types.candidate_pow_receipt_root_records import (
+    CandidatePoWReceiptRootRecord,
+)
+
+
+def test_defaults(sample_candidate_pow_receipt_root_record_params):
+    candidate_pow_receipt_roots = CandidatePoWReceiptRootRecord(
+        **sample_candidate_pow_receipt_root_record_params
+    )
+    assert candidate_pow_receipt_roots.candidate_pow_receipt_root == \
```

Comment:
We've avoided this type of line continuation across the codebase and I'd like to continue that.  Can you either make this line long and `noqa` it or some other alternate way to shorten this.